LICENSE

```diff
@@ -1,6 +1,6 @@
   ilastik: interactive learning and segmentation toolkit
 
-      Copyright (C) 2011-2014, the ilastik developers
```

Comment:
Do we really need date ranges in copyright notices?

Some people [suggest](https://softwareengineering.stackexchange.com/questions/130478/what-copyright-date-for-an-update-to-an-open-source-project-from-last-year) just leaving the original publication date.

Reply:
Propose update for now and discuss it on ilastik meeting. It should be consistent across licenses in ilastik files